Automattic/woocommerce-payments
# Add cli npm command

#### Changes proposed in this Pull Request

Simply introduce the `npm run cli` command to be able to easily run commands inside Docker from the development environment - the same one we have on our server.

#### Testing instructions

* Run `npm run cli`, and you should end up with a shell inside the Docker container for WooPayments.

-------------------


**Post merge**
